src/unum/devtelemetry/dt_collector.c

```diff
@@ -786,14 +794,16 @@ int devtelemetry_init(int level)
 {
     if(level == INIT_LEVEL_DEVTELEMETRY)
     {
-#ifndef FEATURE_LAN_ONLY
+        // ToDo: Below FEATURE_FESTATS_ONLY needs to be removed
+        // This is a workaround to support festats on repeater mode 
+#if !defined(FEATURE_LAN_ONLY) && !defined(FEATURE_FESTATS_ONLY)
```

Comment:
Let's not do this yet. It has to be under FEATURE_GUEST_NAT and that feature has to cover a bit more.

Reply:
This is a workaround until guest nat comes. Without this, repeater's devtelemetry does n't work until guest nat comes-in. Anyhow I have removed it. But just giving headsup.